skohub.shacl.ttl

```diff
@@ -0,0 +1,219 @@
+    sh:message "Scope Note should be provided with a language tag." ;
+  ] ;
+  sh:property [
+    sh:path skos:note ;
+    sh:severity sh:Violation ;
+    sh:datatype rdf:langString ;
+    sh:message "Note should be provided with a language tag." ;
+  ] ;
+  sh:property [
+    sh:path skos:notation ;
+    sh:severity sh:Violation ;
+    sh:datatype xsd:string ;
+    sh:message "Notation should be provided as a string." ;
+  ] ;
+  sh:property [
+    sh:path skos:example ;
+    sh:severity sh:Violation ;
+    sh:datatype rdf:langString ;
+    sh:message "Example should be provided with a language tag." ;
+  ] ;
+  sh:property [
+    sh:path skos:narrower ;
+    sh:class skos:Concept ;
+    sh:severity sh:Violation ;
+    sh:message "Target class of skos:narrower should be a skos:Concept" ;
+  ] ;
+  sh:property [ 
+    sh:class skos:Concept ;
+    sh:path skos:narrowerTransitive ;
+    sh:severity sh:Violation ;
+    sh:message "Target class of skos:narrowerTransitive should be a skos:Concept" ;
+  ] ;
+  sh:property [
+    sh:path skos:narrowMatch ;
+    sh:nodeKind sh:IRI ;
+    sh:severity sh:Violation ;
+    sh:message "Target class of skos:narrowMatch should be a skos:Concept (IRI)" ;
+  ] ;
+  sh:property [
+    sh:path skos:broader ;
+    sh:class skos:Concept ;
+    sh:severity sh:Violation ;
+    sh:message "Target class of skos:broader should be a skos:Concept" ;
+  ] ;
+  sh:property [
+    sh:path skos:broaderTransitive ;
+    sh:class skos:Concept ;
+    sh:severity sh:Violation ;
+    sh:message "Target class of skos:broaderTransitive should be a skos:Concept" ;
+  ] ;
+  sh:property [
+    sh:path skos:broadMatch ;
+    sh:nodeKind sh:IRI ;
+    sh:severity sh:Violation ;
+    sh:message "Target class of skos:broadMatch should be a skos:Concept" ;
+  ] ;
+  sh:property [
+    sh:path skos:related ;
+    sh:class skos:Concept ;
+    sh:severity sh:Violation ;
+    sh:message "Target class of skos:related should be a skos:Concept" ;
+  ] ;
+  sh:property [
+    sh:path skos:relatedMatch ;
+    sh:nodeKind sh:IRI ;
+    sh:severity sh:Violation ;
+    sh:message "Target class of skos:relatedMatch should be a skos:Concept (IRI)" ;
+  ] ;
+  sh:property [
+    sh:path skos:closeMatch ;
+    sh:nodeKind sh:IRI ;
+    sh:severity sh:Violation ;
+    sh:message "Target class of skos:closeMatch should be a skos:Concept (IRI)" ;
+  ] ;
+  sh:property [
+    sh:path skos:exactMatch ;
+    sh:nodeKind sh:IRI ;
+    sh:severity sh:Violation ;
+    sh:message "Target class of skos:exactMatch should be a skos:Concept (IRI)" ;
+  ] ;
+  sh:property [
+    sh:path skos:inScheme ;
+    sh:class skos:ConceptScheme ;
+    sh:severity sh:Violation ;
+    sh:message "Target class of skos:inScheme should be a skos:ConceptScheme" ;
+  ] ;
+  sh:property [
+    sh:path skos:hasTopConcept ;
+    sh:class skos:ConceptScheme ;
+    sh:severity sh:Violation ;
+    sh:message "Target class of skos:hasTopConcept should be a skos:ConceptScheme" ;
```

Comment:
```suggestion
    sh:class skos:Concept ;
    sh:severity sh:Violation ;
    sh:message "Target class of skos:hasTopConcept should be a skos:Concept" ;
```

Reply:
it should have been `skos:topConceptOf` anyway.